mindjunkies/courses/migrations/0011_coursetoken.py

```
@@ -0,0 +1,25 @@
+
+    dependencies = [
+        ('courses', '0010_enrollment_payment_status_enrollment_transaction_id'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CourseToken',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('status', models.CharField(choices=[('pending', 'Pending'), ('accepted', 'Accepted'), ('running', 'Running')], default='pending', max_length=10)),
```

Ruff (0.8.2)

Line too long (163 > 119)

(E501)